### PR TITLE
chore: add Account Management FAQs

### PR DESCRIPTION
**Pull request by filipelima18**
Preview [here](https://nc93vh-3000.csb.app/faq). Closes PC-1953.

### CODE REVIEW

File: packages/projects-docs/pages/faq.mdx [hunk 1]
```diff
@@ -241,6 +234,148 @@ The application uses a Node.js port that hasn’t been upgraded to Node.js 14 as
...
+1. Click the "[Invite members](https://codesandbox.io/t/overview)" link shown at the left sidebar of your [Dashboard](https://codesandbox.io/dashboard/).
+2. Input the username or email you wish to add in the input box.
+3. Select the user role.
+4. Click "Add member".
+
+The user will receive an email invitation and in-app notification asking them to confirm.
+
+## How can I leave or delete a workspace?
+
+You can leave or delete a workspace from the "Overview" tab of your [Workspace Settings](https://codesandbox.io/t/overview), by clicking on "Leave workspace", or "Delete workspace", respectively.
```
Comment (necoline) [inline packages/projects-docs/pages/faq.mdx:268]:
```suggestion
You can leave or delete a workspace from the "Overview" tab of your [Workspace Settings](https://codesandbox.io/t/overview), by clicking on "Leave workspace", or "Delete workspace", respectively.

There are some cases where leaving a workspace is not permitted.
1. You are the only workspace member with admin rights: Every workspace requires at least one admin member. Before you can leave the workspace, you must assign a new admin. 

2. You are trying to leave the only workspace connected to your account: Every account must be associated with at least one workspace. If you want to leave the only workspace connected to your account, you may create a new workspace and then leave the workspace you want to be disassociated with. Alternatively, deleting your account will remove you from any workspaces that you are a member of. 
```

File: packages/projects-docs/pages/faq.mdx [hunk 2]
```diff
@@ -241,6 +234,148 @@ The application uses a Node.js port that hasn’t been upgraded to Node.js 14 as
...
+
+First, check if the payment method associated with your subscription has expired or is facing any issues.
+
+You can check or edit your payment methods from the "Overview" tab of your [Workspace Settings](https://codesandbox.io/t/overview) by clicking on "Edit billing information", under "CodeSandbox Pro".
+
+If everything seems to be in order but you're still having issues with the payment, fill out our [Support form](https://codesandbox.io/support) by picking "Pro Subscriptions" and "Payment Problems". This will ensure we prioritize your request.
+
+## Can I change my subscription?
+
+Yes, you can upgrade or downgrade your plan at any time.
```
Comment (necoline) [inline packages/projects-docs/pages/faq.mdx:341]:
```suggestion
Yes, you can upgrade or downgrade your plan at any time from the settings page. 

```

**Review by necoline — CHANGES_REQUESTED** [review]
Looks great! I made suggestions for two of the answers

**Comment by filipelima18** [comment]
Thanks for the suggestions @necoline! Committed & ready for review

**Review by necoline — APPROVED** [review]

**Pull request merged**